collectors/python.d.plugin/am2320/metadata.yaml

```
@@ -12,43 +12,98 @@ meta:
       list: []
   info_provided_to_referring_integrations:
     description: ''
-  keywords: []
+  keywords:
+    - temperature
+    - am2320
+    - sensor
+    - humidity
   most_popular: false
 overview:
   data_collection:
     metrics_description: 'Monitor AM2320 metrics with Netdata for optimal temperature and humidity sensor performance. Improve your sensor performance with comprehensive dashboards and anomaly detection.'
-    method_description: ''
+    method_description: 'This collector will try to contact an AM2320 sensor over i2c to retrieve temperature and humidity values.'
```

Sounded a bit like it was talking about autodetection

```suggestion
    method_description: 'It retrieves temperature and humidity values by contacting an AM2320 sensor over i2c.'
```